### ISSUE TITLE
abnormal hair follicle regression 

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/mammalian-phenotype-ontology#2949]
Consider placing abnormal hair follicle regression MP:0003875 as a direct child term of abnormal hair cycle catagen phase MP:0008859. Any reason why these terms are currently siblings?

Add the following new synonyms to existing term "accelerated hair follicle regression MP:0000392"
accelerated hair follicle regression MP:0000392
DEF: earlier onset of catagen phase during the cyclic transformation of the hair follicle
NEW SYN: early onset of hair follicle regression
NEW SYN: premature hair follicle regression
NEW SYN: accelerated catagen
NEW SYN: accelerated entry into catagen
NEW SYN: precocious catagen development
NEW SYN: premature onset of catagen
(Reminder to self): Annotation in J:61509

Add new terms:

failure of hair follicle regression (a child of abnormal hair follicle regression MP:0003875)
DEF: failure to enter and/or complete the regression phase of the hair cycle (catagen) during which cell proliferation ceases, the hair follicle shortens, and an anchored club hair is produced
SYN: failure of catagen entry
SYN: failed catagen
SYN: failure of hair follicle involution
Annotations in J:183487

delayed hair follicle regression (a child of abnormal hair follicle regression MP:0003875)
DEF: late onset of the initiation of the catagen phase during the cyclic transformation of the hair follicle
SYN: delayed catagen onset
SYN: catagen retardation
SYN: delayed catagen entry
SYN: delayed catagen development
SYN: delayed catagen initiation

**Comment by sbello** [comment]
Looking at the annotations to abnormal regression most refer to either failure to enter or exit catagen phase. And the annotations to  MP:0008859 are much the same. Looking at the definition of hair follicle regression it specifically refers to the time for the onset of catagen. The problem with obsoleting the regression term is it is often due to prolonged anagen phase. I think we may be better served making delayed exit from anagen (MP:0010137) a child of the regression term And adding the 'delayed hair follicle regression' and additional synonyms as synonyms to this term (since delayed entry into catagen is equivalent to delayed exit from anagen). I would then add delayed exit from catagen to both catagen and regression.
For the failure term I would add this but if you add the synonyms add this as narrow synonyms. It might be better to add the failed onset term separately as a child of failed regression rather than adding the narrow synonyms. I'm not sure we want to add the failure term under catagen phase since failure to enter catagen could also be considered failure to exit anagen. 
Basically I think we need to keep regression as a sibling to anagen and catagen since abnormalities in regression maybe the result of abnormalities in exiting anagen or the catagen phase.
